conjure_oxide/src/parse.rs

```diff
@@ -247,12 +248,16 @@ fn parse_vec_op(
 fn parse_constant(constant: &serde_json::Map<String, Value>) -> Option<Expression> {
     match &constant["Constant"] {
         Value::Object(int) if int.contains_key("ConstantInt") => {
-            Some(Expression::Constant(Constant::Int(
-                int["ConstantInt"].as_array()?[1]
-                    .as_i64()?
-                    .try_into()
-                    .unwrap(),
-            )))
+            Some(Expression::Constant(
+                Metadata::new(),
+                Constant::Int(
+                    //TODO (kf77): Get metadata from JSON?
```

Comment:
I don't think there is anything sensible to extract from the JSON as metadata. I feel we will need a native parser soon (not that soon, maybe next year)